core/domain/subtopic_page_domain_test.py

```diff
@@ -218,6 +218,56 @@ def test_update_written_translations(self):
         self.assertEqual(
             self.subtopic_page.to_dict(), expected_subtopic_page_dict)
 
+    def test_cannot_create_subtopic_page_change_with_invalid_change_dict(self):
```

Comment:
Rename: `*_with_empty_dict` for a more explicit description (all of the dicts you test here are "invalid"... what makes this one special?)

Reply:
done